Cargo.toml

```diff
@@ -50,9 +59,17 @@ tokio = { version = "1.25.0", features = ["full"] }
 pretty_assertions = "1.0.0"
 
 [features]
-default = ["html", "rust"]
+default = ["html", "tree-sitter-all", "rust"]
 rust = []
-html = ["html-escape", "comrak", "handlebars", "syntect"]
+html = ["html-escape", "comrak", "handlebars", "tree-sitter-highlight"]
+tree-sitter-all = ["tree-sitter-javascript", "tree-sitter-typescript", "tree-sitter-json", "tree-sitter-regex", "tree-sitter-css", "tree-sitter-toml", "tree-sitter-md"]
+tree-sitter-javascript = ["dep:tree-sitter-javascript"]
+tree-sitter-typescript = ["dep:tree-sitter-typescript"]
+tree-sitter-json = ["dep:tree-sitter-json"]
+tree-sitter-regex = ["dep:tree-sitter-regex"]
+tree-sitter-css = ["dep:tree-sitter-css"]
+tree-sitter-toml = ["dep:tree-sitter-toml"]
+tree-sitter-md = ["dep:tree-sitter-md"]
```

Comment:
seems unecessary to have

Reply:
Just always enable them all? Ok.

Reply:
yea, we dont need to conditionally have them in any of our scenarios